cluster/centos/node/scripts/kubelet.sh

```diff
@@ -14,12 +14,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
 MASTER_ADDRESS=${1:-"8.8.8.18"}
 NODE_ADDRESS=${2:-"8.8.8.20"}
 DNS_SERVER_IP=${3:-"192.168.3.100"}
 DNS_DOMAIN=${4:-"cluster.local"}
-
+KUBECONFIG_PATH=${KUBECONFIG_PATH:-/opt/kubernetes/cfg}
```

Comment:
I see we are off in centos land... nevermind if that is where they are putting stuff.